api/tests/integration/domain/services/pick-certification-challenge_test.js

```diff
@@ -0,0 +1,603 @@
+const { expect, databaseBuilder, airtableBuilder } = require('../../../test-helper');
+const cache = require('../../../../lib/infrastructure/caches/learning-content-cache');
+const placementProfileService = require('../../../../lib/domain/services/placement-profile-service');
+const certificationChallengesService = require('../../../../lib/domain/services/certification-challenges-service');
+const moment = require('moment');
+const { PIX_COUNT_BY_LEVEL } = require('../../../../lib/domain/constants');
+
+describe('Integration | CertificationChallengeService | pickCertificationChallenge', function() {
+  const placementDate = new Date('2020-01-01T00:00:00Z');
+  const certificationDate = _addOneDayToDate(placementDate);
+  const sufficientPixValueToBeCertifiableOnCompetence = PIX_COUNT_BY_LEVEL;
+  const unsufficientPixValueToBeCertifiableOnCompetence = 1;
+  let certifiableUserId;
+
+  beforeEach(() => {
+    certifiableUserId = databaseBuilder.factory.buildUser().id;
+  });
+
+  afterEach(() => {
+    airtableBuilder.cleanAll();
+    return cache.flushAll();
+  });
+
+  it('picks no unoperative challenges', async () => {
```

Comment:
C'est possible de mettre plutôt les tournures en positif ? 
```suggestion
  it('picks only operative challenges', async () => {
```